config/initializers/sidekiq.rb

```diff
@@ -6,6 +6,7 @@
 end
 
 Sidekiq.configure_server do |config|
+  config.log_formatter = Sidekiq::Logger::Formatters::JSON.new
```

Comment:
I'd have thought you'd not need this: https://github.com/alphagov/govuk_sidekiq/blob/77298fcf54a66514a384e34a35b643be3c16d2d3/lib/govuk_sidekiq/sidekiq_initializer.rb#L15

Reply:
oops, good shout